### DISCUSSION TITLE
Falha ao enviar o email

### DISCUSSION THREAD

**Discussion by leoothomazz** [balta-io/2811#2] [Q&A] [answered]
Unable to read data from the transport connection: The connection was closed.

Boa tarde professor, estou recebendo este erro ao tentar enviar o email.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @leoothomazz isto ocorre por conta da sua conexão com seu servidor de E-mail ser rejeitada.
Ou está usando um endereço SMTP inválido ou a porta...
